app/views/shared/schools/mentors/_can_not_remove.html.erb

```diff
@@ -6,6 +6,10 @@
   <%= t(".title") %>
 </label>
 
-<p class="govuk-body">
   <%= simple_format(t(".description", mentor_name: mentor.full_name)) %>
-</p>
+
+  <ul class="govuk-list govuk-list--bullet">
+    <% mentor.placements.where(school:).find_each do |placement| %>
```

Comment:
After actioning the above comment, this could probably benefit moving into a decorator.